site/content/hero.mdx

```diff
@@ -0,0 +1,17 @@
+---
+description: Featured content hero
+title: Hero
+---
+
+<Hero title="Crossroads">
```

Comment:
Hero title should be "ComposeUI" instead.

Reply:
Fixed.